stripper/ze_eternal_journey_go_v1_4t9.cfg

```diff
@@ -0,0 +1,13 @@
+;Fix Heal burning human for the entire of Heal's lifetime which could cause 1 hp players to die
+modify:
+{
+	match:
+	{
+		"classname" "trigger_hurt"
+		"targetname" "heal_str_hurt"
+	}
+	delete:
+	{
+		"OnStartTouch" "!activatorIgniteLifetime00-1"
```

Comment:
This output is incorrect, you've got 2 extra escape characters in here. Make sure you're copying the output as it's seen in the map.